client/asset/eth/nodeclient.go

```diff
@@ -500,6 +500,11 @@ func (n *nodeClient) isRedeemable(secretHash [32]byte, secret [32]byte, contract
 	})
 }
 
+// sendSignedTransaction injects a signed transaction into the pending pool for execution.
+func (n *nodeClient) sendSignedTransaction(ctx context.Context, tx *types.Transaction) error {
+	return n.ec.SendTransaction(ctx, tx)
```

Comment:
This can now be `n.leth.ApiBackend.SendTx(ctx, tx)`, but it looks like the client method gives some extra logging.  Still looking into it.

Reply:
Please don't use the `ethclient.Client`. It's an unnecessary encode-decode for both the transaction and the result. It's going to `n.leth.ApiBackend.SendTx(ctx, tx)` in the end anyway.